app.py

```diff
@@ -16,10 +26,54 @@
 
 db = SQLAlchemy(app, session_options={"autocommit": True})
 
-@app.route("/")
+login_manager= LoginManager(app)
+login_manager.init_app(app)
+login_manager.login_view= "login"
+
+@login_manager.user_loader
+def load_user(user_id):
+    return Users.query.get(int(user_id))
+
+@app.route("/", methods=["GET", "POST"])
 def index():
-    return 
+    return flask.render_template("signup.html")
+
+
+@app.route("/signup", methods=["GET", "POST"])
+def signup():
+    if flask.request.method == "GET":
+        return flask.render_template("signup.html")
+    if flask.request.method == "POST":
+        emailData = flask.request.form["email"] 
```

Comment:
For email we need to check if it has the "@" and the top level domain is after it. We also need to check if fields contains any empty spaces.